api/util/errorLog-function.js

```diff
@@ -9,13 +9,13 @@ const ErrorLog = require('../models/ErrorLog');
  * @param {string} req.body.apiEndpoint - The location of the error
  * @param {string} req.body.errordescription - The description of the error
  */
-async function addErrorLog(req){
+async function addErrorLog(userEmail, errorTime, apiEndpoint, errorDescription){
   let errorSaved = true;
   const newError = new ErrorLog({
-    userEmail: req.body.userEmail,
-    errorTime: req.body.errorTime,
-    apiEndpoint: req.body.apiEndpoint,
-    errorDescription: req.body.errorDescription
+    userEmail: userEmail,
```

Comment:
fun fact, in javascript doing 
```suggestion
    userEmail,
```
implies
```js
userEmail: userEmail
```

when the left and right side have the same name. Change all of them to the suggested format